tools/redhat/setup.py

```diff
@@ -2,12 +2,9 @@
     i.e. https://access.redhat.com/security/data/csaf/v2/advisories/2024/rhsa-2024_4546.json
 """
 
-from setuptools import setup, find_packages
+from setuptools import setup
 
-REQUIRES = [
-    "jsonschema",
-    "requests"
-]
+REQUIRES = ["jsonschema", "requests"]
```

Comment:
I think you need to add `urllib3` (or `packageurl-python`) here?

Reply:
Ok, updated that too, thanks.